nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/webapp/css/header.css

```diff
@@ -34,12 +34,13 @@ md-toolbar.md-small .md-toolbar-tools {
 }
 
 #header .icon {
-    font-size:30px;
+    font-size:32px;
     color: #004849;
+    padding-left:6px
```

Comment:
@scottyaslan looks like the 6px padding-left was not enough to center the icons within each button. I must have misjudged it originally. Changing it to 12px works for me now.